api/src/routers/meals.js

```diff
@@ -1,8 +1,31 @@
 import express from "express";
 import knex from "../database_client.js";
+import dayjs from "dayjs";
 
 const meals = express.Router();
 
+meals.post("/add", async (req, res) => {
+  try {
+    const { title, description, location, when, max_reservations, price, img_url } = req.body;
+    const formattedDate = dayjs(when).format("YYYY-MM-DD HH:mm:ss");
+    // Insert new meal into the database
+    await knex("Meal").insert({
```

Comment:
If you instead only wrap this statement in the try catch you can communicate to the user that the request failed because we did not succeed in adding the request to the database and/or log a more helpful error message

Reply:
So as I understand, I should make two different try and catchs for both formatteddate and knex insert data , so it would be more visible where is the mistake spesificly if there is an error. Is that correct?

Reply:
Correct: If the request is bad: we should tell the client(400), if the database fails, then it is our fault (500)